docs/discussion/values/community-values.rst

```diff
@@ -0,0 +1,31 @@
+Community Values
+===================
+
+This section will be about the values that the ``ppb`` community should strive
+for.
+
+
+Values
+------
+
+We have a Code of Conduct marking the minimum buy in to ``ppb`` culture,
+but these values are aspirational and goals for the community.
+
+Radical Acceptance
++++++++++++++++++++
+
+This is about diversity. The author of ``ppb`` is a white trans lesbian with a
+non-traditional education background. We welcome everyone from everywhere. We
+want you involved.
+
+No Contempt Culture
+++++++++++++++++++++
+
+Python for game development is scoffed at by many professionals. This culture of
+contempt is often discouraging for new developers who find a tool they love only
+to listen to how bad it is. In pursuit of radical acceptance, we want to avoid
+contempt culture.
+
+Strong opinions are fine. Every member of the core developers has strong
+opinions on software. Being opinionated is appreciated, but don't share your
+contempt for given software or tools here.
```

Comment:
Do we want links to this? https://blog.aurynn.com/2015/12/16-contempt-culture/ is the traditional one, but there's potentially other good ones.